media-sound/teamspeak-server-bin/files/teamspeak.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=TeamSpeak3 Server
+After=network.target
+
+[Service]
+Type=simple
+Environment=LD_LIBRARY_PATH=/opt/teamspeak3-server
+WorkingDirectory=/opt/teamspeak3-server
+ExecStartPre=/bin/rm -f /dev/shm/7gbhujb54g8z9hu43jre8
+ExecStart=/opt/bin/ts3server inifile=/etc/teamspeak3-server/ts3server.ini
+ExecStop=/bin/kill -15 $MAINPID
+StandardOutput=syslog
```

Comment:
Why are you forcing syslog? I think most systemd users would prefer it to respect their default.

Reply:
This are the systemd units, which are currently in portage. I've took a look into the docs, so I will remove it completly, to use the systemds default, whatever the user uses.